### PR TITLE
add warning when podcast does not load

### PR DESCRIPTION
**Pull request by waqasnasir**
add warning message when iframe does not load.

### CODE REVIEW

**Comment by kartikcho** [comment]
Hi @waqasnasir! It looks like your implementation of the twitter component's warning message doesn't seem to work. You should give it another shot and let's get this merged soon :)

File: src/containers/podcast/Podcast.js [hunk 1]
```diff
@@ -3,10 +3,21 @@ import "./Podcast.css";
 import { podcastSection } from "../../portfolio";
 import { Fade } from "react-reveal";
 
+const cantDisplayError = "<div class='centerContent'><h2>Can't load? Check privacy protection settings</h2></div>";
+
+function timeOut(){
```
Comment (kartikcho) [inline src/containers/podcast/Podcast.js:8]: The Twitter message implementation uses [Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html) so you should try to do something similar here too with a fallback loader.

**Review by kartikcho — CHANGES_REQUESTED** [review]
@saadpasta what's up with the Vercel deployments asking for permissions?

File: src/containers/podcast/Podcast.js (hunk 1 above)
Comment (naveen521kk) [reply src/containers/podcast/Podcast.js:8]: Actually speaking it uses lazy loading, https://reactjs.org/docs/react-api.html#reactlazy, which in turn uses suspense.  See #181

**Comment by naveen521kk** [comment]
Any update on this @waqasnasir

**Comment by kartikcho** [comment]
@naveen521kk what do you think about closing this to allow new contributors to work on this issue? There seems to be no activity for almost a month on this PR.

**Comment by naveen521kk** [comment]
> what do you think about closing this to allow new contributors to work on this issue? There seems to be no activity for almost a month on this PR.

Ok @kartikcho. If @waqasnasir wants to works on this again he can make a new PR.

**Pull request closed without merging**